examples/portfolio/src/components/MainHead.astro

```diff
@@ -9,5 +9,6 @@ const { title = 'Jeanine White: Personal Site', description = 'The personal site
 <title>{title}</title>
 
 <link rel="icon" type="image/x-icon" href="/favicon.ico" />
-<link rel="preconnect" href="https://fonts.gstatic.com" />
-<link href="https://fonts.googleapis.com/css2?family=Inter:wght@200;400;700;900&display=swap" rel="stylesheet" />
+<link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100&display=swap" rel="stylesheet"> 
```

Comment:
Any reason this particular line changed from `200;400;700;900` => `100`?

Reply:
> Any reason this particular line changed from `200;400;700;900` => `100`?

oh sorry that was a copy & paste mistake